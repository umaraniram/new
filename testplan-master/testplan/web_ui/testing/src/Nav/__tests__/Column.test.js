import React from 'react';
import {shallow} from 'enzyme';
import {StyleSheetTestUtils} from "aphrodite";

import Column from '../Column';

describe('Column', () => {
  beforeEach(() => {
    // Stop Aphrodite from injecting styles, this crashes the tests.
    StyleSheetTestUtils.suppressStyleInjection();
  });

  afterEach(() => {
    // Resume style injection once test is finished.
    StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
  });

  it('shallow renders the correct HTML structure', () => {
    const column = shallow(
      <Column width={"20em"} >
        <p className='unique' />
      </Column>
    );
    expect(column).toMatchSnapshot();
  });
});
