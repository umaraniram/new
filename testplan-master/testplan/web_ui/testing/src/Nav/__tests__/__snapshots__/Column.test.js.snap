// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column shallow renders the correct HTML structure 1`] = `
<Fragment>
  <div
    className="column_1e6n0oe-o_O-leftColumn_1c7wfp"
  >
    <p
      className="unique"
    />
  </div>
  <div
    className="column_1e6n0oe-o_O-rightColumn_1bksxyz"
    onMouseDown={[Function]}
  >
    <div
      className="column_1e6n0oe-o_O-splitter_11em7wq"
    />
  </div>
</Fragment>
`;
